src/run.rs

```diff
@@ -95,12 +111,40 @@ pub fn image_ids() -> io::Result<HashSet<String>> {
             output
                 .lines()
                 .map(|line| line.trim().to_owned())
-                .filter(|line| !line.is_empty())
+                .filter_map(|line| {
+                    if line.is_empty() {
+                        let tab_index = line.find('\t').unwrap();
+                        let (image_id, date_str) = line.split_at(tab_index);
+                        Some((image_id.to_owned(), parse_docker_date(date_str.trim())))
+                    } else {
+                        None
+                    }
+                })
                 .collect()
         })
         .map_err(|error| io::Error::new(io::ErrorKind::Other, error))
 }
 
+// Extract a Duration from Docker's non-standard date format used in `docker image ls`
+// Example input: "2017-12-20 16:30:49 -0500 EST"
+fn parse_docker_date(date_str: &str) -> Duration {
+    let date_fmt = "%Y-%m-%d %H:%M:%S %z";
+    // Chrono can't read the "EST", so remove it before parsing
+    let timezone_index = 25;
```

Comment:
This hardcoded index seems brittle. Could we instead use something like `rsplitn` to remove it?